common/lib/port_config.py

```diff
@@ -67,6 +67,21 @@ def gnmi_ctl_set(self, params):
                 raise ExecuteCMDException(f"Failed to execute command {cmd}")
             log.passed(f"{cmd}")
             return output
+        
+        def gnmi_ctl_set_invalid_args(self, params):

```

Comment:
you dont need this. this is the reverse of the existing one. the utils function is sufficient with the pass/fail checks reversed.